initfiles/sbin/hpcc-run.sh.in

```diff
@@ -52,134 +52,143 @@ getIPS(){
 }
 
 getDali(){
-        DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
+    DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
 }
 
 createIPListFile(){
-    _file=$1
-    echo "$IPS" > $_file
+    local _output=$1
+    echo "$IPS" > $_output
 }
 
 createIPListFileExcludeDIP(){
-  _file=$1
-  echo "$IPS" | grep -v $DIP  > $_file
+    local _input=$1
+    local _output=$2
```

Comment:
actually doing 

```bash
grep -v -w "$DIP" $_input > $_output
```

seems to work fine so long as the values in $DIP are each on a new line